cogs/urbandict_utils/urbandictpages.py

```diff
@@ -0,0 +1,42 @@
+import discord
+import re
+from discord.ext.commands import Context
+from cogs.utils.paginator import Pages
+import typing as t
```

Comment:
```suggestion
import re
import typing as t

import discord
from discord.ext.commands import Context

from cogs.utils.paginator import Pages
```
These imports need to be sorted